pallets/asset/src/lib.rs

```diff
@@ -1200,13 +1201,16 @@ impl<T: Config> Module<T> {
     /// Ensure `ticker` is fully printable ASCII (SPACE to '~').
     fn ensure_ticker_ascii(ticker: &Ticker) -> DispatchResult {
         let bytes = ticker.as_slice();
+        let s = String::from_utf8_lossy(bytes);
+        let result = s.chars().all(char::is_alphanumeric);
```

Comment:
This check will reject trailing NULL bytes, which we want to allow.  The alphanumeric check should be done in the look below.